src/Service/VendorService/UploadService/UploadServiceVendorService.php

```diff
@@ -79,13 +84,56 @@ public function load(): VendorImportResultMessage
 
             try {
                 $item = $this->store->move($item->getId(), self::DESTINATION_FOLDER.'/'.$identifier);
+                $state = VendorState::INSERT;
+                $this->statsLogger->info($this->getVendorName().' image moved', [
+                    'service' => self::class,
+                    'type' => $type,
+                    'identifier' => $identifier,
+                    'url' => $item->getUrl(),
+                ]);
             } catch (\Exception $e) {
-                // The image may have been moved to we ignore this error an goes to the next item.
-                continue;
+                if (preg_match('/^to_public_id(.+)already exists$/', $e->getMessage())) {
+                    $item = $this->store->move($item->getId(), self::DESTINATION_FOLDER.'/'.$identifier, true);
+                    $state = VendorState::UPDATE;
+                    $this->statsLogger->info($this->getVendorName().' image updated', [
+                        'service' => self::class,
+                        'type' => $type,
+                        'identifier' => $identifier,
+                        'url' => $item->getUrl(),
+                    ]);
+                } else {
+                    // The image may have been moved to we ignore this error an goes to the next item.
+                    $this->statsLogger->info($this->getVendorName().' error moving image', [
```

Comment:
Should this only log as `info` or as `error`

Reply:
This should be `warning` and the other in 127 should be error